Replicator/Replicator.cc

```diff
@@ -514,7 +426,7 @@ namespace litecore { namespace repl {
 
 
     // Start off by getting the local checkpoint, if this is an active replicator:
-    void Replicator::getLocalCheckpoint() {
+    void Replicator::getLocalCheckpoint(bool noRemote) {
```

Comment:
Since there's only one existing caller of `getLocalCheckpoint`, I think it'd be clearer to remove the `getRemoteCheckpoint` call from the end of this method, and add it to there. Then you don't need to add a parameter. (It's already kind of weird that I had `getLocalCheckpoint` get the remote checkpoint too. No idea why I did that...)